screen-releases/src/main/java/com/ivy/releases/ReleasesScreen.kt

```diff
@@ -0,0 +1,200 @@
+package com.ivy.releases
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyListScope
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material3.ElevatedButton
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedCard
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalUriHandler
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.ivy.navigation.Navigation
+import com.ivy.navigation.navigation
+
+@Composable
+fun ReleasesScreenImpl() {
+    val viewModel: ReleasesViewModel = viewModel()
+    val uiState = viewModel.uiState()
+
+    ReleasesUi(
+        uiState = uiState,
+        onEvent = {
+            viewModel.onEvent(it)
+        }
+    )
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun ReleasesUi(
+    uiState: ReleasesState,
+    onEvent: (ReleasesEvent) -> Unit
+) {
+    val nav = navigation()
+
+    Scaffold(
+        topBar = {
+            TopAppBar(
+                title = {
+                    TopAppBarTitle(title = "Releases")
+                },
+                navigationIcon = {
+                    BackButton(nav = nav)
+                }
+            )
+        }
+    ) { innerPadding ->
+        LazyColumn(
+            modifier = Modifier.padding(start = 12.dp, end = 12.dp, bottom = 12.dp),
+            verticalArrangement = Arrangement.spacedBy(8.dp),
+            contentPadding = innerPadding,
+        ) {
+            content(
+                releasesState = uiState,
+                onEvent = onEvent
+            )
+        }
+    }
+}
+
+@Composable
+private fun TopAppBarTitle(title: String) {
+    Text(
+        text = title,
+        maxLines = 1,
+        overflow = TextOverflow.Ellipsis
+    )
+}
+
+@Composable
+private fun BackButton(nav: Navigation) {
```

Comment:
Don't pass `nav: Navigation` like this. Instead use:
```
val nav = navigation()
```

inside the BackButton.